Source/WebKit/UIProcess/Cocoa/UserMediaCaptureManagerProxy.cpp

```diff
@@ -232,9 +233,9 @@ class UserMediaCaptureManagerProxySourceProxy final
         if (!m_settings) {
             m_settings = m_source->settings();
             if (m_widthConstraint || m_heightConstraint) {
-                auto desiredSize = m_source->computeResizedVideoFrameSize({ m_widthConstraint, m_heightConstraint }, m_source->intrinsicSize());
+                auto desiredSize = m_source->computeResizedVideoFrameSize({ m_widthConstraint, m_heightConstraint }, protectedSource()->intrinsicSize());
```

Comment:
Ditto here, let's take protectedSource local variable for `m_source->computeResizedVideoFrameSize`, `intrinscSize` and `videoFrameRotation`

Reply:
Fixed!